dangerfile.js

```diff
@@ -131,30 +131,25 @@ function git(args) {
 
     if (packagesToShow.includes('react')) {
       const reactProd = results.find(
-        r => r.bundleType === 'UMD_PROD' && r.packageName === 'react'
+        r =>
+          r.packageName === 'react' &&
+          (r.prevFileSizeChange !== 0 || r.prevGzipSizeChange !== 0)
```

Comment:
Can you explain this change? I don't get it.

We were specifically looking for the UMD prod bundle earlier because that's the one we consider the "React size".

Same below.

Reply:
Yeah, I initially thought when looking at that code that a different version of those libs could slip through the net and not raise a warning. But you're right, the highlights are specifically for the prod user-facing builds, so I've reverted the change